### PR TITLE
Update minimal template

### PR DESCRIPTION
**Pull request by meister03**
Important Note: CachePlugin has to be updated on deno.land

### CODE REVIEW

**Comment by itohatweb** [comment]
I'm not sure about the PR title 🤔

**Comment by meister03** [comment]
> I'm not sure about the PR title 🤔

my terminal was lagging :|

**Comment by Yaikava** [comment]
Why did you change it dotenv to use `.env.example`?
That is wrong imo.

**Comment by meister03** [comment]
> Why did you change dotenv to use `.env.example`? That is wrong imo.

it worked with that and people can change it in a sec, when they want  a other name for their env

**Comment by Yaikava** [comment]
> Important Note: CachePlugin has to be updated on deno.land

I think you are supposed to import it from main module.

**Comment by Yaikava** [comment]
> > Why did you change dotenv to use `.env.example`? That is wrong imo.
> 
> it worked with that and people can change it in a sec, when they want a other name for their env

This way secrets can get into git easily by accident. They can change it yeah, but it is bad idea imo.

File: template/minimal/configs.ts [hunk 1]
```diff
@@ -1,6 +1,5 @@
 import { dotEnvConfig } from "./deps.ts";
 
-dotEnvConfig({ export: true });
-
+dotEnvConfig({ export: true, path: './.env.example' });
```
Comment (lts20050703) [inline template/minimal/configs.ts:3]:
```suggestion
dotEnvConfig({ export: true });
```

Comment (lts20050703) [reply template/minimal/configs.ts]: Don't use .env.example

**Comment by lts20050703** [comment]
@meister03 Please run `deno fmt`

**Review by Skillz4Killz — APPROVED** [review]

**Pull request merged**